xpk/xpk.py

```diff
@@ -473,6 +473,13 @@ def add_env_config(args):
               'environment, a value must be specified.'
           )
           env[variable] = os.environ[variable]
+
+  if args.debug_dump_gcs:
+    if 'XLA_FLAGS' in env:
+      # Caveat: might be overwriting something passed by user
+      print('overwriting XLA_FLAG from env-file')
```

Comment:
Nit 'xpk_print'

Reply:
Thanks, changed to raising an error as per Rafi's suggestion